src/libraries/System.Threading.ThreadPool/tests/WindowsThreadPool/System.Threading.ThreadPool.WindowsThreadPool.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <!-- This test project is Windows only as it uses the Windows Threadpool -->
+    <TargetFramework>$(NetCoreAppCurrent)-windows</TargetFramework>
+    <TestRuntime>true</TestRuntime>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\ThreadPoolTests.cs" />
+    <Compile Include="..\RegisteredWaitTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="$(CommonTestPath)System\IO\TempFile.cs"
+             Link="CommonTest\System\IO\TempFile.cs" />
+    <Compile Include="$(CommonTestPath)System\Threading\ThreadTestHelpers.cs"
+             Link="CommonTest\System\Threading\ThreadTestHelpers.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <RuntimeHostConfigurationOption Include="System.Threading.ThreadPool.UseWindowsThreadPool" Value="true" />
```

Comment:
Maybe you can also explicitly set this to false in System.Threading.ThreadPool.Tests.csproj to get test coverage for the portable threadpool in native AOT.

Reply:
Or is this PR switching the default threadpool to use for native AOT on Windows? I am not able to tell.

Either way, we should make sure that one of threadpool implementations is trimmed for native AOT by default.

Reply:
I'm setting it explicitly to false in System.Threading.ThreadPool.Tests.csproj for now. My intention is to leave the default threadpool for Native AOT unchanged but I believe my changes make the portable threadpool the default one for every runtime. By any chance you know what file should be updated to make Native AOT use the Windows threadpool by default?

Reply:
Here is one way to do it:

- Add `<UseWindowsThreadPool Condition="$(UseWindowsThreadPool) == ''">false</UseWindowsThreadPool>` here: https://github.com/dotnet/runtime/blob/84a7be769fa9054dac7a973dda538cf61a488e56/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets#L46
and
- Add `<IlcArg Condition="'$(UseWindowsThreadPool)' != ''" Include="--feature:System.Diagnostics.Debugger.IsSupported=$(UseWindowsThreadPool)" /> here: https://github.com/dotnet/runtime/blob/84a7be769fa9054dac7a973dda538cf61a488e56/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets#L259

Reply:
Thanks, however to keep using the windows threadpool on Native AOT the first one should be true right?
`<UseWindowsThreadPool Condition="$(UseWindowsThreadPool) == ''">true</UseWindowsThreadPool>`

Reply:
Right